src/style.css

```diff
@@ -1,3 +1,75 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family is set globally, which is fine, but ensure that the 'Roboto' font is correctly loaded in your HTML file. This is already done in your HTML file using Google Fonts, so this line is correctly implemented.